### ISSUE TITLE
I sent it and never saw it again

### ISSUE THREAD

**Issue by guvenkab1** [toncenter/ton-wallet#105]
I sent 21.95 Ton coins using the correct address and memo for my FTX wallet. I have done this many time, but my coins are still not there 21 hours later. I see posts mentioning similar issues please inform us...

![Adsız](https://user-images.githubusercontent.com/101662799/158437456-a773b346-10fb-4ef3-92ad-c2a807f32b31.png)

**Comment by Chisdan** [comment]
could you solve it? i have the same issue

**Comment by guvenkab1** [comment]
Yep. Check the transection history, if transaction show as complete you should open a ticket to FTX. They need some proof, some secreen shots and they let your Ton's to your account.

**Comment by guvenkab1** [comment]
![image](https://user-images.githubusercontent.com/101662799/165186486-95dbbec8-ba2e-4994-90f0-1afea6334c19.png)

**Comment by Chisdan** [comment]
i used OKX, but here is my problem 
![imagen](https://user-images.githubusercontent.com/104396704/165196349-59a8ebe5-7c6f-4a72-a43c-c5fb8e6cb229.png)
depsit suspended by token migration.
will ask to OKS support.
thanks for your answer

**Comment by Chisdan** [comment]
> Yep. Check the transection history, if transaction show as complete you should open a ticket to FTX. They need some proof, some secreen shots and they let your Ton's to your account.

do tou know if i can see the transaction from ton-wallet web? because in OKX history doesn´t appears, cos it never start due to suspended deposits.
thanks
